ui/src/services/allocation.js

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 
 class AllocationService {
-  BASE_URL = process.env.BASE_URL || 'http://localhost:9090/model';
+  BASE_URL = process.env.BASE_URL || '{PLACEHOLDER_BASE_URL}';
```

Comment:
Hey @AjayTripathy I've updated the function call to now check for the PLACEHOLDER value and override with a default if it is not set.

I originally wanted to do something like:

`BASE_URL = process.env.BASE_URL || '{PLACEHOLDER_BASE_URL}' || 'http://localhost:9090/model';`

but realized that if the placeholder is not overwritten by the script, it will always be picked up as a valid value.  

I don't like that the conditional is checked _every_ API call, but it's definitely not a huge slowdown. 

My qeustion to you (and any react experts): is there a way to run this check _once_ upon initializing the component, rather than doing it every time we want to make an API call?